doc_src/gear_developers/executor_pool.md

```diff
@@ -92,8 +92,14 @@ thus simultaneous task executions are capped by the number of processes in each
 
 ## Managing capacity of executor pools
 
-- As of this writing, capacities of executor pools are centrally controlled by the antikythera team.
-  Please contact the antikythera team to request a limit increase.
+- Capacities of executor pools are determined by [AntikytheraCore.ExecutorPool.Setting](https://github.com/access-company/antikythera/blob/master/core/executor_pool/setting.ex).
```

Comment:
Is it intentional that you make a link to the source code instead of the module document?
I think referring the module document is sufficient here.

```suggestion
- Capacities of executor pools are determined by `AntikytheraCore.ExecutorPool.Setting`.
```

Reply:
I should not have added an unnecessary link.
Thank you.